blocklist_user.c

```diff
@@ -0,0 +1,90 @@
+#include <linux/netlink.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <unistd.h>
+
+#define MAX_PAYLOAD 1024
+#define LINE_LENGTH 20
+
+int main(int argc, char *argv[])
+{
+    /* Read interface pair from blocklist.txt */
```

Comment:
Add an option to specify block list. If no option is passed, just display the status of vwifi driver.